priv/xcodegen/Sources/TemplateApp/ContentView.swift

```diff
@@ -8,9 +8,85 @@ import LiveViewNative
 
 struct ContentView: View {
     var body: some View {
-        LiveView(.automatic(
-            development: .localhost(path: "%LVN_PREFERRED_ROUTE%"),
-            production: .custom(URL(string: "https://%LVN_PREFERRED_PROD_URL%%LVN_PREFERRED_ROUTE%")!)
-        ))
+        #LiveView(
+            .automatic(
+                development: .localhost(path: "%LVN_PREFERRED_ROUTE%"),
+                production: URL(string: "https://%LVN_PREFERRED_PROD_URL%%LVN_PREFERRED_ROUTE%")!
+            ),
+            addons: []
+        ) {
```

Comment:
The template uses the `#LiveView` macro with an empty `addons` parameter by default now.